artistools/lightcurve.py

```diff
@@ -752,6 +759,36 @@ def make_viewing_angle_peakmag_delta_m15_scatter_plot(modelnames, key, colours,
     plt.close()
 
 
+def calculate_costheta_phi_for_viewing_angles(viewing_angles):
+    costheta_viewing_angle_bins = ['-1.0 \u2264 cos(\u03B8) < -0.8', '-0.8 \u2264 cos(\u03B8) < -0.6',
+                                   '-0.6 \u2264 cos(\u03B8) < -0.4', '-0.4 \u2264 cos(\u03B8) < -0.2',
+                                   '-0.2 \u2264 cos(\u03B8) < 0', '0 \u2264 cos(\u03B8) < 0.2',
+                                   '0.2 \u2264 cos(\u03B8) < 0.4', '0.4 \u2264 cos(\u03B8) < 0.6',
+                                   '0.6 \u2264 cos(\u03B8) < 0.8', '0.8 \u2264 cos(\u03B8) < 1']
+    phi_viewing_angle_bins = ['0 \u2264 \u03D5 < \u03c0/5', '\u03c0/5 \u2264 \u03D5 < 2\u03c0/5',
+                              '2\u03c0/5 \u2264 \u03D5 < 3\u03c0/5', '3\u03c0/5 \u2264 \u03D5 < 4\u03c0/5',
+                              '4\u03c0/5 \u2264 \u03D5 < \u03c0', '9\u03c0/5 < \u03D5 < 2\u03c0',
+                              '8\u03c0/5 < \u03D5 \u2264 9\u03c0/5', '7\u03c0/5 < \u03D5 \u2264 8\u03c0/5',
+                              '6\u03c0/5 < \u03D5 \u2264 7\u03c0/5', '\u03c0 < \u03D5 \u2264 6\u03c0/5']
+    for i in viewing_angles:
+        if i < 10:
+            costheta_index = 0
+            phi_index = i
+            print(str(i) + "   " + costheta_viewing_angle_bins[costheta_index] + "   " +
+                  phi_viewing_angle_bins[phi_index])
+
+        else:
+            split_viewing_angle = [int(d) for d in str(i)]
+
+            costheta_index = split_viewing_angle[0]
+            phi_index = split_viewing_angle[1]
+
+            print(str(i) + "   " + costheta_viewing_angle_bins[costheta_index] + "   " +
+                  phi_viewing_angle_bins[phi_index])
```

Comment:
This can be simplified to:
```suggestion
        MABINS = 100
        phibins = int(math.sqrt(MABINS))
        costheta_index = i // phibins
        phi_index = i % phibins

        print(f"{i:4d}   {costheta_viewing_angle_bins[costheta_index]}   {phi_viewing_angle_bins[phi_index]}")
```